src/tracks/SampleTrack.cpp

```diff
@@ -109,6 +109,20 @@ SampleTCO::SampleTCO( Track * _track ) :
 
 
 
+SampleTCO::SampleTCO(const SampleTCO& orig) :
+	SampleTCO(orig.getTrack())
+{
+	// TODO: This creates a new SampleBuffer for the new TCO, eating up memory
+	// & eventually causing performance issues. Letting tracks share buffers
+	// when they're identical would fix this, but isn't possible right now.
+	delete m_sampleBuffer;
+	m_sampleBuffer = new SampleBuffer(*orig.m_sampleBuffer);
```

Comment:
```suggestion
	*m_sampleBuffer = *orig.m_sampleBuffer;
```
This saves an unnecessary deallocation and allocation. Not too important though - this bit of code isn't performance-critical, and will probably change when `SampleBuffer`s can be shared.

Reply:
Wait... doesn't this go back to sharing the sample buffers? Or is this some weird syntax for delete, copy, assign?

Reply:
Never mind, dereference both buffers => assignment from buffer to buffer => copy assignment called.